Ambit/Source/Ambit/Mode/GltfExporterExternal.h

```diff
@@ -0,0 +1,54 @@
+//   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License").
+//   You may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+#pragma once
+
+#include "Exporters/GLTFLevelExporter.h"
+#include "Serialization/BufferArchive.h"
+
+#include "GltfExporterExternal.generated.h"
+
+UINTERFACE()
+class UGltfExporterExternal : public UInterface
```

Comment:
what is the meaning of this "External"?

Reply:
External is being used for any 3P code we were relying on. I could also change this to `Wrapper` if that simplifies it.